packages/server/src/config/default.ts

```diff
@@ -5,9 +5,20 @@ const config: AppConfig = {
     redisPort: process.env.REDIS_PORT
         ? parseInt(process.env.REDIS_PORT, 10)
         : 6379,
-    redisDataCache: {
+    memoizerRedis: {
         enabled: true,
     },
+    uniswap: {
```

Comment:
would it make more sense to have `network` as the top level and platform specific within the respective networks?

Reply:
Yeah I think something like `networks.mainnet.uniswapV3` as a config path might make more sense, since it mirrors the route structure

Reply:
Hmm I can see that working, but it makes it tougher to fetch component specific config. For ex, to get all the uniswap config you'd have to 

```js
Object.keys(config.networks).map(network => ({ [network]: config.networks[network].prop }));
```

Let's refactor later when we get need to add more network config.

Reply:
👍